doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -171,6 +171,10 @@ Bluetooth samples
 
   * The Bluetooth 3-wire coex sample because of the removal of the 3-wire implementation.
 
+* :ref:`direct_test_mode` sample:
+
+  * Fixed a compilation warning when used with minimal pinout Skyworks FEM.
```

Comment:
This is a bit vague. Is the warning different after this fix or is it not shown anymore?

Reply:
It is not shown anymore